spec/support/env_helpers.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module EnvHelpers
   def use_fake_connection
     allow(ENV).to receive(:[]).with('FAKE_QUEUE').and_return(true)
```

Comment:
if you removed the need for `FAKE_QUEUE` is this helper still needed?

Reply:
Nope.  I will dump this file.